Composer/packages/extensions/visual-designer/src/components/renderers/ElementWrapper.tsx

```diff
@@ -21,7 +21,7 @@ const nodeBorderSelectedStyle = css`
 
 // BotAsks, UserAnswers and InvalidPromptBrick nodes selected style
 const nodeBorderDoubleSelectedStyle = css`
-  outline: 2px solid #0078d4;
+  border: 2px solid #0078d4;
```

Comment:
The problem is not entirely fixed. We used `outline` here because of we didn't want to infect the box size.
If you keep clicking between two nodes, you will find the element floating problem.

Reply:
Reference this demo: http://jsfiddle.net/f8q6axgg/
![image](https://user-images.githubusercontent.com/8528761/75154678-3c0dfc00-5749-11ea-8188-f4fb4ab2f883.png)

It should be helpful.

Reply:
Thanks, It's fixed with `box-shadow`